Backend/Models/DTO/UserDTO.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Backend.Models.Database;
+using Newtonsoft.Json;
+using System;
+
+namespace Backend.Models.DTO
+{
+    public class UserDTO
+    {
+        [JsonProperty("uid")]
+        public int Uid { get; set; }
+
+        [JsonProperty("name")]
+        public string Name { get; set; }
+
+        [JsonProperty("surname")]
+        public string Surname { get; set; }
+
+        [JsonProperty("registrationDate")]
+        public DateTime RegistrationDate { get; set; }
+
+        [JsonProperty("dismissalDate")]
+        public DateTime? DismissalDate { get; set; }
+
+        [JsonProperty("position")]
+        public string Position { get; set; }
+
+        [JsonProperty("state")]
+        public string State { get; set; }
```

Comment:
Maybe it would be more clear to specify, what state this is? In this case, it is most likely state of employment, so `EmploymentState` would be more clear. Technically, user could have more states in the future.

Reply:
Fixed